troupon/static/scss/partials/_typography.scss

```diff
@@ -43,3 +43,30 @@ h1 {
     margin-right: 5px;
   }
 }
+
+.fa-edit {
+  padding-left: 8px;
+}
+
+.item-info {
+  .title, .advertiser, .price, .address {
+    padding-bottom: 20px;
+  }
+
+  address {
+    margin-bottom: 0px

```

Comment:
Declaration should be terminated by a semicolon<br>`0px` should be written without units as `0`